### ISSUE TITLE
Unable to set Device admin

### ISSUE THREAD

**Issue by dolezel** [oddmouse/cordova-plugin-locktask#3]
When I run the command `adb shell dpm set-device-owner my.package/.MySubclass` I get the error

```
Error: Unknown admin: ComponentInfo{my.package/my.package.MySubclass}
```

Do you know what I'm doing wrong?

**Comment by dolezel** [comment]
Sorry, got it.
MySubclass have to be in same folder/package as `MainActivity.java`

**Comment by jweidner-mbible** [comment]
Double check what your DeviceAdminReceiver.getComponentName() method is actually returning.   In my case, I hadn't realized that my app's gradle configuration was setting a different applicationId for the "productFlavor" I was building.  After I added a log statement to see the real component name, I then knew what I needed to specify in the set-device-owner command.

Log.d(TAG,"componentName="+DeviceAdminReceiver.getComponentName(mApplicationContext));
